pages/reference/OS/advanced.md

```diff
@@ -109,26 +106,31 @@ dtoverlay=i2c-rtc,ds1307
 dtoverlay=lirc-rpi
 ```
 
-This parsing will only be done if the value is a valid string; so if it doesn't begin with a quote `"`, the value will be parsed as a single string and not split into several lines. For instance `RESIN_HOST_CONFIG_dtoverlay = i2c-rtc,ds1307` will translate to:
+This parsing will only be done if the value is a valid string, so if it doesn't begin with a quote `"`, the value will be parsed as a single string and not split into several lines. For instance `RESIN_HOST_CONFIG_dtoverlay = i2c-rtc,ds1307` will translate to:
 
 ```
 dtoverlay=i2c-rtc,ds1307
 ```
 
-### Disabling rainbow splash screen
+### Disabling the rainbow splash screen
 
 To disable the Raspberry Pi rainbow splash screen, add the `disable_splash=1` entry to `config.txt`.
 
 __Note:__ This setting disables the Raspberry Pi rainbow splash screen but does not disable the {{ $names.company.lower }} logo splash screen. If you would like to replace the {{ $names.company.lower }} logo with your custom splash logo, replace `splash/resin-logo.png` located in the [boot partition][boot-partition] of the image.
 
 ### Further Reading
 
-There are more details on the options available in `config.txt` over at
-[elinux's RPi Config page][elinux].
+There are more details on the options available in `config.txt` over at [elinux's RPi Config page][elinux].
 
-[elinux]:http://elinux.org/RPiconfig
+[boot-partition]:/reference/OS/overview/2.x/#image-partition-layout
+[cli]:/reference/cli/reference/balena-cli/#envs
+[config-txt]:https://www.raspberrypi.org/documentation/configuration/config-txt/
 [config-vars]:/learn/manage/configuration
+[config-vars-fleet]:/learn/manage/configuration/#managing-fleet-configuration-variables
+[config-vars-device]:/learn/manage/configuration/#managing-device-configuration-variables
+[device-provisioning]:/learn/welcome/primer/#device-provisioning
+[device-tree-overlay]:https://github.com/raspberrypi/linux/blob/rpi-4.1.y/arch/arm/boot/dts/overlays/README
```

Comment:
`rpi-4.1.y` is very old - this probably ought to be updated to https://github.com/raspberrypi/linux/blob/rpi-4.19.y/arch/arm/boot/dts/overlays/README ? (or whatever kernel branch it is that BalenaOS uses)